server/common/health.py

```diff
@@ -0,0 +1,31 @@
+from http import HTTPStatus
+from flask import make_response, jsonify
+
+from server import __version__ as cellxgene_version
+from server.common.data_locator import DataLocator
+
+
+def _is_accessible(path):
+    if path is None:
+        return True
+
+    try:
+        dl = DataLocator(path)
+        print("exists", dl.exists())
```

Comment:
extra print statement should be removed.

Reply:
doh!  will do